### ISSUE TITLE
Crash when plugins replace sign blocks during events

### ISSUE THREAD

**Issue by dktapps** [pmmp/PocketMine-MP#6041]
Link to crashdump: https://crash.pmmp.io/view/9184390

### Additional comments
Seems like the block was set during PlayerInteractEvent?

**Comment by AzaleeX** [comment]
Yes, I have the impression that this happens when the **`PlayerInteractEvent`** is called more frequently when you `click on a sign`.
